redfish-core/lib/ethernet.hpp

```diff
@@ -1945,6 +2267,34 @@ inline void requestEthernetInterfacesRoutes(App& app)
                 return;
             }
         }
+
+        if (oemObject)
+        {
+            std::optional<nlohmann::json> openbmc;
+            if (!json_util::readJson(*oemObject, asyncResp->res, "OpenBMC",
+                                     openbmc))
+            {
+                BMCWEB_LOG_ERROR << "Property Value Format Error ";
```

Comment:
upstream these should be warnings since client error.. I don't see these being that useful, so I vote warning here too

Reply:
done